packages/playground/src/weblets/profile_manager.vue

```diff
@@ -794,9 +805,9 @@ async function storeAndLogin() {
     await activate(mnemonic.value, keypairType.value);
   } catch (e) {
     if (e instanceof TwinNotExistError) {
-      isNonActiveMnemonic.value = true;
       openAcceptTerms.value = true;
       termsLoading.value = true;
+      isNonActiveMnemonic.value = true;
```

Comment:
is there a reason for this change ?

Reply:
yes, it's turn ```shouldActivateAccount``` to true which fire ```activateAccount``` function

Reply:
I mean, why we move it down? it was already on 3 lines above, its fine but i just want to know if there is a logical reason

Reply:
to fire createAccount function after accepting terms.